src/DiffOp/Diff/Diff.php

```diff
@@ -366,11 +356,31 @@ public function toArray( $valueConverter = null ) {
 	}
 
 	/**
+	 * @since 2.0
+	 *
+	 * @param mixed $target
+	 *
+	 * @return bool
+	 */
+	public function equals( $target ) {
```

Comment:
You could also implement this by doing a diff between the two diffs and seeing if it is equal to an empty diff.

![inception](https://cloud.githubusercontent.com/assets/146040/6631508/bf53a9bc-c927-11e4-946e-3c99a4dd94bc.gif)